# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sharpconstruct/sharpconstruct-0.12-gcc43.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- sharpconstruct-0.12/include/DataDir.hh.orig	2011-04-17 18:26:10.148077818 +0200
+++ sharpconstruct-0.12/include/DataDir.hh	2011-04-17 18:27:34.028077967 +0200
@@ -19,6 +19,7 @@
 #ifndef DATADIR_HH
 #define DATADIR_HH
 
+#include <cstdlib>		// free
 #include <glibmm/ustring.h>
 
 #ifdef WIN32
